third-party/bettertls/README.md

```diff
@@ -0,0 +1,17 @@
+# BetterTLS Test Suite
+
+Generated using the Netflix [bettertls] project.
+
+[bettertls]: https://github.com/Netflix/bettertls
+
+## Pathbuilding 
+
+To regenerate pathbuilding test data:
+
+1. Install Go
+2. Generate the JSON testdata export for the path building suite:
+
+```bash
+GOBIN=$PWD go install github.com/Netflix/bettertls/test-suites/cmd/bettertls@latest
+./bettertls export-tests --suite pathbuilding --out ./pathbuilding.tests.json
+```
```

Comment:
I really wanted to add a CI task that would use a fresh Go install + checkout of the project to verify there's no diff between the vendored `test.json` and the one produced by the clean checkout but unfortunately the upstream project regenerates all of the certificates each time and so there's _always_ a massive diff. 

I think short of investing a bunch of work in the upstream project we'll have to do our best to keep this in-sync manually. The upstream project seems pretty sleepy so I don't suspect there will be a lot of new tests added over time.